File: src/main/java/me/soo/helloworld/model/user/UserFindPasswordRequest.java
package me.soo.helloworld.model.user;

import lombok.AllArgsConstructor;
import lombok.Getter;

import javax.validation.constraints.Email;
import javax.validation.constraints.NotBlank;

@Getter
@AllArgsConstructor
public class UserFindPasswordRequest {

    @NotBlank(message = "아이디를 입력하세요.")
    private final String userId;

    @Email(message = "이메일을 형식에 맞게 입력해주세요")
    @NotBlank
    private final String email;
}

File: src/main/java/me/soo/helloworld/model/user/UserFindPasswordRequest.kt
package me.soo.helloworld.model.user

import javax.validation.constraints.Email
import javax.validation.constraints.NotBlank

data class UserFindPasswordRequest(

    @field:NotBlank(message = "아이디를 입력하세요.")
    val userId: String,

    @field:Email(message = "이메일을 형식에 맞게 입력해주세요")
    @field:NotBlank
    val email: String
)